apollo-router/src/plugins/traffic_shaping/deduplication.rs

```diff
@@ -72,7 +72,7 @@ where
     }
 
     async fn dedup(
```

Comment:
@BrynCooke Look in detail

Reply:
I think Dedupe is OK. We can tackle the performance issues it causes in a follow-up